lighthouse-core/gather/driver.js

```diff
@@ -423,23 +346,36 @@ class Driver {
     });
   }
 
+  /**
+   * Alias for 'sendCommandToSession(method, undefined, ...params)'
+   * @template {keyof LH.CrdpCommands} C
+   * @param {C} method
+   * @param {LH.CrdpCommands[C]['paramsType']} params
+   * @return {Promise<LH.CrdpCommands[C]['returnType']>}
+   */
+  sendCommand(method, ...params) {
+    return this.sendCommandToSession(method, undefined, ...params);
+  }
+
   /**
    * Call protocol methods.
    * @private
    * @template {keyof LH.CrdpCommands} C
    * @param {C} method
+   * @param {string=} sessionId
    * @param {LH.CrdpCommands[C]['paramsType']} params
    * @return {Promise<LH.CrdpCommands[C]['returnType']>}
    */
-  _innerSendCommand(method, ...params) {
+  _innerSendCommand(method, sessionId, ...params) {
     const domainCommand = /^(\w+)\.(enable|disable)$/.exec(method);
-    if (domainCommand) {
+    // TODO: store domain state for each session.
```

Comment:
does this actually need to be done? is this a TODO before landing or a note that we don't support this right now?

Reply:
I think our domain state management is not useful. But there's no reason to have it for the main session and not the other sessions, so for that reason it needs to be done. Forgot about it :)

Reply:
oh, the jsdoc says it's to avoid users "stomping on each other". ok that makes more sense. I don't think we have any gatherers that enable/disable stuff for a child target, so it's not 100% necessary, but will do now anyway.